### PR TITLE
DOC Document deprecations related to CMSMain refactor

### PR DESCRIPTION
**Pull request by GuySartorelli**
## Issue
- https://github.com/silverstripe/silverstripe-cms/issues/2947

### CODE REVIEW

File: en/08_Changelogs/5.4.0.md [hunk 1]
```diff
@@ -210,6 +210,52 @@ The `SilverStripe\CMS\Model\SiteTree.DESCRIPTION` localisation key (along with t
...
+- [`CMSMain::LinkPreview()`](api:SilverStripe\CMS\Controllers\CMSMain::LinkPreview()) has been deprecated. Use [`SiteTree::CMSEditLink()`](api:SilverStripe\CMS\Model\SiteTree::CMSEditLink()) instead.
+- [`CMSMain::SiteTreeAsUL()`](api:SilverStripe\CMS\Controllers\CMSMain::SiteTreeAsUL()) has been deprecated. Use [`CMSMain::TreeAsUL()`](api:SilverStripe\CMS\Controllers\CMSMain::TreeAsUL()) instead.
+- [`CMSMain::getSiteTreeFor()`](api:SilverStripe\CMS\Controllers\CMSMain::getSiteTreeFor()) has been deprecated. Use [`CMSMain::getTreeFor()`](api:SilverStripe\CMS\Controllers\CMSMain::getTreeFor()) instead.
+- [`CMSMain::CanOrganiseSitetree()`](api:SilverStripe\CMS\Controllers\CMSMain::CanOrganiseSitetree()) has been deprecated. Use [`CMSMain::canOrganiseTree()`](api:SilverStripe\CMS\Controllers\CMSMain::canOrganiseTree()) instead.
+- [`CMSMain::getSearchContext()`](api:SilverStripe\CMS\Controllers\CMSMain::getSearchContext()) has been deprecated. Use [`SiteTree::getDefaultSearchContext()`](api:SilverStripe\CMS\Model\SiteTree::getDefaultSearchContext()) instead.
```
Comment (emteknetnz) [inline en/08_Changelogs/5.4.0.md:236]: Remember to update this based on https://github.com/silverstripe/silverstripe-cms/pull/3036/files#r1877297991

Comment (emteknetnz) [inline en/08_Changelogs/5.4.0.md]: Remember to update this based on https://github.com/silverstripe/silverstripe-cms/pull/3036/files#r1877297991

**Review by emteknetnz — CHANGES_REQUESTED** [review]
Just one small change

I have manually checked all the entries here (took a while :-]) and everything looks good

File: en/08_Changelogs/5.4.0.md (hunk 1 above)
Comment (ChloeSartorelli) [reply en/08_Changelogs/5.4.0.md]: Updated

**Review by emteknetnz — APPROVED** [review]

**Pull request merged**